packages/server-wallet/src/config.ts

```diff
@@ -39,6 +41,8 @@ const config: ServerWalletConfig = {
   rpcEndpoint: process.env.RPC_ENDPOINT,
   chainNetworkID: process.env.CHAIN_NETWORK_ID || '0x0',
   ethAssetHolderAddress: process.env.ETH_ASSET_HOLDER_ADDRESS,
+  erc20Address: process.env.ERC20_ADDRESS,
```

Comment:
I don't think it makes sense to have something called `erc20Address` / `ERC20_Address`. This information will probably need to be a mapping of some kind? For example, DAI, MKR, and USDC all have different addresses.